smartmon.sh

```diff
@@ -22,6 +22,9 @@ $1 ~ /^ *[0-9]+$/ && $2 ~ /^[a-zA-Z0-9_-]+$/ {
 SMARTCTLAWK
 )"
 
+# Find the smartctl binary, usually in /usr/sbin (Linux) or /usr/local/sbin (FreeBSD)
+SMARTCTL_BIN="$(whereis smartctl | awk '{print $2}')"
```

Comment:
Wouldn't this essentially just be the same as omitting the path prefix from the `smartctl` commands, and letting the shell's PATH find it?